db/connection.rb

```diff
@@ -1,10 +1,10 @@
 require 'pg'
 require 'active_record'
 
-ActiveRecord::Base.establish_connection(
-  :adapter => "postgresql",
-  :database => "landlord"
-)
+ActiveRecord::Base.establish_connection({
+  database: 'landlord',
+  adapter: 'postgresql',
+})
```

Comment:
This change is equivalent to the original.  The hash is implied.  The order of the keys is insignificant in a hash.